packages/@okta/vuepress-site/architecture-center/reference-architectures/directory-coexistence/index.md

```diff
@@ -22,13 +21,46 @@ Use directory coexistence for:
 * Consolidating multiple Identity Providers for different applications and services into one provider.
 * Merging different identity stores into a single Identity Provider after merging with or acquiring another company.
 
+## Associated Case Studies
+
+<figure class="main-card">
+  <img
+    class="main-card__image"
+    src="/img/architecture/thumbs/companyx.jpg"
+    alt="CompanyX"
+  >
+  <figcaption class="main-card__body">
+    <h2 class="main-card__title">
+      <a
+        class="main-card__main-link"
+        href="/architecture-center/case-studies/companyx/"
+      >
+        User Migration for Company X
+      </a>
+    </h2>
+    <div class="main-card__wrapper">
+      <span class="main-card__tag">
+        Case Study
+      </span>
+      <span class="main-card__tag">
+        User Migration
+      </span>
+      <span class="main-card__tag">
+        Directory Coexistence
+      </span>
+    </div>
+    <p class="main-card__text">
+      Learn how Company X applied the directory coexistence reference architecture when planning their user migration strategy and the benefits they saw. <a class="main-card__link" href="/architecture-center/case-studies/companyx/">Learn more.</a>
+    </p>
+  </figcaption>
+</figure>
+
 ## Lab
 
-* [Overview](/docs/reference/architecture-tutorials/directory-coexistence/lab-overview)
-* [Prerequisites](/docs/reference/architecture-tutorials/directory-coexistence/lab-prerequisites)
-* Tutorial 1: [Migrate users from Azure Active Directory to Universal Directory](/docs/reference/architecture-tutorials/directory-coexistence/lab-1-azure-ad)
-* Tutorial 2: [Migrate users from an on-premises LDAP directory server to Universal Directory](/docs/reference/architecture-tutorials/directory-coexistence/lab-2-ldap-server)
-* Tutorial 3: [Migrate users from an on-premises generic database to Universal Directory](/docs/reference/architecture-tutorials/directory-coexistence/lab-3-generic-database)
+* [Overview and prerequisites](/architecture-center/reference-architectures/directory-coexistence/lab-overview)
+* Tutorial 1: [Migrate users from Azure Active Directory to Universal Directory](/architecture-center/reference-architectures/directory-coexistence/lab-1-azure-ad)
```

Comment:
I'm not convinced that losing the "to Universal Directory" is beneficial here.

Reply:
The `to UD` content adds complexity to the headings. Cutting it makes for a more concise heading. The [Lab Intro](https://64396364338d1005f3493f56--reverent-murdock-829d24.netlify.app/architecture-center/reference-architectures/directory-coexistence/) makes the Universal Directory aspect clear, and it can be mentioned in the intro for each tutorial.